src/pages/admin/reports/components/report-analytics/ReportAnalytics.module.scss

```
@@ -5,3 +5,23 @@
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 20px;
+    padding: 20px 0;
+}
+
+.footer-button {
+    display: flex;
+    align-items: center;
+    @include type.subtitle-1-medium;
+    gap: 20px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)